src/components/ReviewPage.jsx

```diff
@@ -33,10 +49,56 @@ export default function ReviewPage() {
 				<div className="side-buttons">
 					<span className="username">@{review.owner} </span>
 					<span> votes:{review.votes} </span>
-					<button onClick={handleUpVoteClick}>upVote!</button>
-					<button onClick={handleDownVoteClick}>downVote :(</button>
+					<button onClick={handleReviewUpVoteClick}>upVote!</button>
+					<button onClick={handleReviewDownVoteClick}>
+						downVote :(
+					</button>
 				</div>
-			</main>
-		</>
+			</section>
+			<section className="comment-display">
+				<h2 className="comment-display-title">Comments!</h2>
+				{!comments ? (
+					<h2>Comments Loading...!</h2>
+				) : (
+					<ul>
+						{comments.map((comment) => {
```

Comment:
I'd maybe add the date or something so we know we're seeing the most recent comments first